### DISCUSSION TITLE
Can Github code scan be enabled for Azure DevOps git repo?

### DISCUSSION THREAD

**Discussion by soumyadipmitralpl** [community/community#25462] [Actions]
<p>My company is using Azure DevOps git repos for version control. We want to leverage the new Github code scanning functionality for those repos. How can we enable code scanning using CodeQL engine for our Azure git repos?</p>
<p>Also is there any documentation available for integrating the code scanning to our build pipeline too using the Azure git repos?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/soumyadipmitralpl">@soumyadipmitralpl</a>,<br>
Search in the <a href="https://marketplace.visualstudio.com/azuredevops" rel="noopener nofollow ugc">Visual Studio Marketplace</a>, do not find any Azure DevOps extensions for CodeQL.</p>
<p>According to the introduction from the documentation “<a href="https://lgtm.com/help/lgtm/about-lgtm" rel="noopener nofollow ugc"><strong>About LGTM</strong></a>”, it seems that currently CodeQL is not supported on Azure DevOps.<br>
The current supported repository hosts are below:</p>
<ul>
<li>Bitbucket Cloud</li>
<li><a href="http://GitHub.com" rel="noopener nofollow ugc">GitHub.com</a></li>
<li><a href="http://GitLab.com" rel="noopener nofollow ugc">GitLab.com</a></li>
</ul>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/0/5082c233ceacebfedb0597dfdb87f86080324013.png" data-download-href="/uploads/short-url/bueluwBvp8P9YmvpusJxqhgzD2P.png?dl=1" title="Supported" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/46307996/181091224-e60446a4-6cc6-4e52-8035-307b686923f1.png" alt="Supported" data-base62-sha1="bueluwBvp8P9YmvpusJxqhgzD2P" width="659" height="499" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/0/5082c233ceacebfedb0597dfdb87f86080324013_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Supported</span><span class="informations">816×619 22.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by soumyadipmitralpl** [comment]
<p>Thank you <a class="mention" href="/brightran">@brightran</a> for your reply.</p>
